docs/index.html

```diff
@@ -96,6 +97,8 @@ <h1>Options explained</h1>
         <dd>(integer) The amount of miliseconds in which the plugin snaps to the desired panel.</dd>
         <dt>easing</dt>
         <dd>(function) An easing function specificing the snapping motion.</dd>
+        <dt>timeout</dt>
+        <dd>(integer) The amount of miliseconds for timeout while slide effect.</dd>
```

Comment:
Does this not read a bit better?
`The amount of miliseconds where no scroll event should occur before the plugin decides to snap to a panel.`

Reply:
Also, seems like I made a typo myself: `miliseconds` > `milliseconds`